openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/SystemRepository.java

```diff
@@ -192,6 +201,8 @@ public void updateSetting(Settings setting) {
         SlackAppConfiguration appConfiguration =
             JsonUtils.convertValue(setting.getConfigValue(), SlackAppConfiguration.class);
         setting.setConfigValue(encryptSlackAppSetting(appConfiguration));
+      } else if (setting.getConfigType() == SettingsType.CUSTOM_UI_THEME_PREFERENCE) {
```

Comment:
can we have this validation for other settings schema as well , there also we might have extra fields coming in

Reply:
for other settings we are reading the Json values , where the validation happens